### PR TITLE
breakpoints-&-router-dom

### PR DESCRIPTION
**Pull request by JaqKent**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
   return (
-    <>
+    <Router>
       <Navbar />
-      <Slider auto={auto} setAuto={setAuto} slides={ITEMS} className={styles.image} />
-      <h1>mundo regalo</h1>
-    </>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/survey" element={<Survey />} />
```
Comment (nathan-hyan) [inline src/App.tsx:12]: Better to put this in a `constants.ts` file and do a .map here, what do you think?

File: src/components/NavBar/index.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import menu from 'assets/menu.svg';
 import logo from 'assets/logoBlanco.png';
 import cart from 'assets/carro.svg';
 import close from 'assets/x.svg';
+import { useNavigate } from 'react-router-dom';
 import { ANIMATION_CONFIG } from './constants';
 import DropsdownMenu from './DropdownMenu';
 import SearchBar from './SearchBar';
```
Comment (nathan-hyan) [inline src/components/NavBar/index.tsx:11]: Sort

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Good stuff

File: src/components/Slider/index.tsx [hunk 3]
```diff
@@ -51,7 +52,7 @@ function Carousel({
         onMouseEnter={() => { setAuto(false); }}
         onMouseLeave={() => { setAuto(true); }}
       >
-        {slideLength >= 2 && (
+        {hide ? '' : slideLength >= 2 && (
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/Slider/index.tsx]: done

File: src/components/Slider/index.tsx [hunk 4]
```diff
@@ -68,7 +69,7 @@ function Carousel({
             )}
           </div>
         ))}
-        {slideLength >= 2 && (
+        {hide ? '' : slideLength >= 2 && (
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/Slider/index.tsx:72]: done

File: src/components/NavBar/index.tsx (hunk 2 above)
Comment (JaqKent) [reply src/components/NavBar/index.tsx:11]: done

File: src/App.tsx (hunk 1 above)
Comment (JaqKent) [reply src/App.tsx:12]: good sugestion

Comment (JaqKent) [reply src/App.tsx]: done

**Pull request merged**